mobile/types.go

```diff
@@ -265,10 +265,11 @@ func (tx *Transaction) GetSigHash() *Hash { return &Hash{tx.tx.SigHash(types.Hom
 func (tx *Transaction) GetCost() *BigInt  { return &BigInt{tx.tx.Cost()} }
 
 func (tx *Transaction) GetFrom(chainID *BigInt) (address *Address, _ error) {
-	if chainID == nil { // Null passed from mobile app
-		chainID = new(BigInt)
+	var signer types.Signer = types.HomesteadSigner{}
+	if chainID != nil {
+	signer = types.NewEIP155Signer(chainID.bigint)
```

Comment:
Usually my IDE is doing these things for me - perhaps I should install one for go ;-)